lms/static/js/edxnotes/views/notes_page.js

```diff
@@ -3,33 +3,54 @@
 define([
     'backbone', 'js/edxnotes/collections/tabs', 'js/edxnotes/views/tabs_list',
     'js/edxnotes/views/tabs/recent_activity', 'js/edxnotes/views/tabs/course_structure',
-    'js/edxnotes/views/tabs/search_results'
+    'js/edxnotes/views/tabs/search_results', 'js/edxnotes/views/tabs/tags'
 ], function (
     Backbone, TabsCollection, TabsListView, RecentActivityView, CourseStructureView,
-    SearchResultsView
+    SearchResultsView, TagsView
 ) {
     var NotesPageView = Backbone.View.extend({
         initialize: function (options) {
+            var scrollToTag, tagsModel;
+
             this.options = options;
             this.tabsCollection = new TabsCollection();
 
-            this.recentActivityView = new RecentActivityView({
+            // Must create the Tags view first to get the "scrollToTag" method.
+            this.tagsView = new TagsView({
                 el: this.el,
                 collection: this.collection,
                 tabsCollection: this.tabsCollection
             });
 
+            scrollToTag = this.tagsView.scrollToTag.bind(this.tagsView);
```

Comment:
I have always used http://underscorejs.org/#bindAll in such situations because then it needs to be done in only one place. Are there any reasons to prefer `bind`?